src/coreclr/vm/riscv64/stubs.cpp

```diff
@@ -1823,19 +1872,18 @@ PCODE DynamicHelpers::CreateDictionaryLookupHelper(LoaderAllocator * pAllocator,
                 indirectionsDataSize += (pLookup->sizeOffset > 2047 ? 4 : 0);
             }
 
-            codeSize += (pLookup->offsets[i] > 2047 ? 16 : 4); // if( > 2047) (8 bytes) else 4 bytes for instructions.
+            codeSize += (pLookup->offsets[i] > 2047 ? 16 : 4); // if( > 2047) (16 bytes) else 4 bytes for instructions.
```

Comment:
in if above size is wrong, it seems that it should be `codeSize += (pLookup->sizeOffset > 2047 ? 24 : 16);`

Reply:
it seems this one is also wrong: 
`codeSize += (pLookup->offsets[i] > 2047 ? 12 : 4); // if( > 2047) (12 bytes) else 4 bytes for instructions.`

Reply:
Done